microsoft-edge/devtools-guide-chromium/progressive-web-apps/simulate-window-controls-overlay.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Simulate PWA Window Controls Overlay
+description: Simulate PWA Window Controls Overlay.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: devtools
+ms.date: 05/13/2024
+---
+# Simulate PWA Window Controls Overlay
+
+In DevTools, in the **Application** tool, you can simulate the Window Controls Overlay feature that Progressive Web Apps (PWAs) have, without having to install the PWA.  The **Application** tool has a **Window Controls Overlay** section under **Manifest**, to display a preview of how your Progressive Web App (PWA) handles window controls, without having to install the PWA and make the styling changes that are necessary to accommodate window controls:
+
+![The 'Window Overlay Controls' section in the Application tool](./simulate-window-controls-overlay-images/pwa-window-controls.png)
+
+1. In the manifest file, set the `display-override` field to `window-controls-overlay`:
+
+   ```json
+   {
+     "display_override": ["window-controls-overlay"]
+   }
+   ```
+
+   For more information, see [Enable the Window Controls Overlay in your app](../../progressive-web-apps-chromium/how-to/window-controls-overlay.md#enable-the-window-controls-overlay-in-your-app) in _Display content in the title bar_.
```

Comment:
I would change this and start by assuming that developers have a PWA that uses the WCO feature already. How about:

```suggestion
1. Open a PWA that uses the WCO feature. For example, open the [1DIV demo app](https://microsoftedge.github.io/Demos/1DIV/dist/) in a new window or tab.

   For more information about using WCO in your PWA, see [Display content in the title bar](../../progressive-web-apps-chromium/how-to/window-controls-overlay.md).
```

Reply:
```
1. Open a Progressive Web App (PWA) that uses the Window Controls Overlay (WCO) feature.
For example, open the [1DIV demo app](https://microsoftedge.github.io/Demos/1DIV/dist/) 
in a new window or tab.

   For more information about using WCO in your PWA, 
   see [Display content in the title bar](../../progressive-web-apps-chromium/how-to/window-controls-overlay.md).
```